### DISCUSSION TITLE
Xdrip and samsung s10+

### DISCUSSION THREAD

**Discussion by AlisaAqua** [NightscoutFoundation/xDrip#2927] [Show and tell]
Hy!
I'm new here and unfortunately a complete newbie with this site and xdrip app.
I have Samsung s10+ and I have tried to install dozen of times xdrip+ but it just won't install itself. I've tried latest official version which was in May, I also tried varanty version 4, and I've also tried latest prerelease version which was released few days ago. Every time I get same result.
First my phone tells me that this is app is made for older phone ver but I still clicked install, the installation was running and then at the end it says that app couldn't be installed.

**Comment by candelaviz** [comment]
I have xdrip installed in samsung S10 (normal not+) since 2019 without problem

**Comment by Navid200** [comment]
@candelaviz 
Please open a new thread and ask any questions you may have.


@AlisaAqua 
Thanks for opening this thread.  Let's go over as few items.

Would you please follow these instructions to install xDrip?
https://navid200.github.io/xDrip/docs/Install.html

If any instructions is not clear, please don't make any assumptions.  In that case, please tell me what is not clear.
Thanks

**Reply by AlisaAqua** [reply]
Guys, this isn't my first time installing apk😅 before I did just like on the link you gave me without success...I swear I did it all like it said. 
But for some reason after like 10th time...it installed itself now...haha!🤦🏻‍♀️😂😂😂
One thing I would like to ask...my doctor mentioned that I need another app in order to use xdrip...but he forgot the name so he couldn't help me much. I'm from Croatia so I'm not sure which of these apps I need to download. OOP2? Or that juggluco?? Or can I use xdrip without using any extra app?? 
Thank you for all your help💗

**Reply by AlisaAqua** [reply]
Disregard the last post...I found out which app..OOP2. 
The xdrip works just fine now!!! 🙂...after going through hell of figuring out how everything needs to be set and how to connect with libre app in settings😅
Thank you again for your help and for all hard work that's been put into creating and tweaking this app. 💗
